### PR TITLE
A new chinese text example on the version 0.8.0

### PR DESCRIPTION
**Pull request by ultimatedaotu**
A new chinese text example based on webwq-search example on the version of jina==0.8.0. Because there are some problems at query-flow, I need your kind help.

### CODE REVIEW

File: zh/textsearch/flow-query.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  chunk_indexer:
+    uses: pods/chunk_indexer/chunk_indexer.yml
+    timeout_ready: 600000
+
+#  ranker:
+#    uses: BM25Ranker
+#
+#  doc_indexer:
+#    uses: pods/doc_indexer/doc_indexer.yml
+#    timeout_ready: 60000
```
Comment (nan-wang) [inline zh/textsearch/flow-query.yml:26]:
```suggestion
  ranker:
    uses: pods/ranker/ranker.yml

  doc_indexer:
    uses: pods/doc_indexer/doc_indexer.yml
```

File: zh/textsearch/pods/chunk_indexer/chunk_indexer.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  - !BinaryPbIndexer
+    with:
+      index_filename: chunk_index.gzip
+    metas:
+      name: chunk_index
+      workspace: $TMP_WORKSPACE
+metas:
+  name: chunk_indexer
+  workspace: $TMP_WORKSPACE
```
Comment (nan-wang) [inline zh/textsearch/pods/chunk_indexer/chunk_indexer.yml:19]:
```suggestion
  workspace: $TMP_WORKSPACE
requests:
  on:
    SearchRequest:
      - !VectorSearchDriver
        with:
          executor: vecidx_index
          traversal_paths: ['c']
      - !ExcludeQL
        with:
          fields:
            - embedding
      - !KVSearchDriver
        with:
          executor: chunk_index
          traversal_paths: ['cm']
    IndexRequest:
      - !VectorIndexDriver
        with:
          executor: vecidx_index
          traversal_paths: ['c']
      - !ExcludeQL
        with:
          fields:
            - embedding
      - !KVIndexDriver
        with:
          executor: chunk_index
          traversal_paths: ['c']
    ControlRequest:
      - !ControlReqDriver {}
```

File: zh/textsearch/pods/doc_indexer/doc_indexer.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+!BinaryPbIndexer
+with:
+  index_filename: doc_index.gzip
+metas:
+  workspace: $TMP_WORKSPACE
```
Comment (nan-wang) [inline zh/textsearch/pods/doc_indexer/doc_indexer.yml:5]:
```suggestion
  workspace: $TMP_WORKSPACE
  name: doc_index
requests:
  on:
    ControlRequest:
      - !ControlReqDriver {}
    SearchRequest:
      - !KVSearchDriver
        with:
          traversal_paths: ['m']
    IndexRequest:
      - !KVIndexDriver
        with:
          traversal_paths: ['r']
```

File: zh/textsearch/pods/encoder/encoder2.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+!RBertEncoder
+metas:
+  py_modules: rbert_encoder.py
+requests:
+  on:
+    [SearchRequest, IndexRequest]:
+      - !EncodeDriver
+        with:
+          method: encode
```
Comment (nan-wang) [inline zh/textsearch/pods/encoder/encoder2.yml:9]:
```suggestion
          traversal_paths: ['c']
```

File: zh/textsearch/pods/ranker/ranker.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+!BM25Ranker
+
+requests:
+  on:
+    SearchRequest:
+      - !Chunk2DocRankDriver
+        with:
+          method: score
```
Comment (nan-wang) [inline zh/textsearch/pods/ranker/ranker.yml:8]:
```suggestion
                    traversal_paths: ['c']
      - !ExcludeQL
        with:
          fields:
            - embedding
            - chunks
          traversal_paths: ['r', 'c']
      - !SliceQL
        with:
          traversal_paths: ['m']
          start: 0
          end: 10
```

File: zh/textsearch/pods/ranker/ranker.yml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+!BM25Ranker
```
Comment (nan-wang) [inline zh/textsearch/pods/ranker/ranker.yml:1]: For `0.8.0`, you will add one line at https://github.com/jina-ai/jina-hub/blob/master/rankers/BM25Ranker/__init__.py#L19

```
required_keys = {'text', 'length'}
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
Thanks for your work. However, this is a misusage of PRs. I've fixed your problems and the main issue is due to the misunderstanding of document recursive structures. You might found more detail at our last EAH. The recording is at https://youtu.be/chmVNrRdiKI?t=2204

**Comment by alexcg1** [comment]
Thanks for your PR @ultimatedaotu ! We're currently restructuring our examples repo, so it might be more useful to post this on our own personal repo and we can link it in our upcoming "Community Examples" section

**Comment by nan-wang** [comment]
@ultimatedaotu We've created an example in Chinese for the `1.0` and please check it out.

**Pull request closed without merging**